frontend/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14.16.1-alpine3.13 as builder
```

Comment:
We should create a task to update this 😬

Reply:
Left a TODO, not brave enough to do it right now